flatdata-generator/flatdata/generator/templates/rust/rust.jinja2

```diff
@@ -8,6 +8,7 @@
 {%- for ns in tree.root.children recursive %}
 {%- if ns | is_namespace %}
 
+#[cfg_attr(rustfmt, rustfmt_skip)]
```

Comment:
This removes a possibility to format the code. Sometimes you want to do this. I would prefer that users specify somewhere outside, if they need it.

Reply:
IMHO: If the code is generated during build, it will not be formatted anyway, if the user checks it in, they can remove this line anyway.

The main reason this is added: Formatting the generated files is slow: Several seconds every time it is saved. It basically forced me to turn of formatting in my editor. Rustfmt seems to be quadratic in file length (or some other sub-optimal algorithms)

Anyway, will remove from this review